kong/plugins/hmac-auth/access.lua

```diff
@@ -199,11 +246,10 @@ local function do_authentication(conf)
   return true
 end
 
-
 function _M.execute(conf)
 
   if ngx.ctx.authenticated_credential and conf.anonymous ~= "" then
-    -- we're already authenticated, and we're configured for using anonymous, 
+    -- we're already authenticated, and we're configured for using anonymous,
```

Comment:
please avoid whitespace changes like this, as it dirties the commit history and makes future archaeological efforts more challenging.